server/plugin.go

```diff
@@ -348,7 +348,7 @@ func (p *Plugin) GetToDo(ctx context.Context, username string, githubClient *git
 		text += fmt.Sprintf("You have %v pull requests awaiting your review:\n", issueResults.GetTotal())
 
 		for _, pr := range issueResults.Issues {
-			text += fmt.Sprintf("* %v\n", pr.GetHTMLURL())
+			text += fmt.Sprintf("* %v - %v\n", pr.GetHTMLURL(), pr.GetTitle())
```

Comment:
why not add the URL as markdown in the title?
`text += fmt.Sprintf("* [%v](%v)\n", pr.GetTitle(),pr.GetHTMLURL())`

Reply:
Good idea!